docs/docs-content/clusters/edge/site-deployment/site-installation/cluster-deployment.md

```diff
@@ -38,8 +38,9 @@ Use the following steps to create a new host cluster so that you can add Edge ho
   [Edge Host Registration](./edge-host-registration.md).
 
 - If you are using more than one Edge host to form a cluster, the hosts in the same cluster must be on the same network.
-
-- You must ensure that the Edge hosts have stable IP addresses. You have the following options to do achieve stable IP
+- 1 IP address is required for the cluster's Virtual IP address (VIP).
```

Comment:
```suggestion
- One IP address is required for the cluster's Virtual IP address (VIP).
```

Reply:
I usually spell out numbers 1 through 10 because, for me, it's easier to read this way :) But this is optional, and if you feel like numbers are better here, so be it 💮

Reply:
We spell them out, 1 - 10. It's in our styleguide 😉